test/test.js

```diff
@@ -48,6 +48,19 @@ $.mockjax( {
 	}
 } );
 
+$.mockjax( {
```

Comment:
noop, every `mockajax` call has a different url, so it will result in different object listener. (see the previous mockajax calls, just before this one)